sources/Adapters/picoTracker/sdcard/sd_card_spi.cpp

```diff
@@ -32,7 +33,9 @@ class RP2040SPIDriver : public SdSpiBaseClass {
 
   void activate() {
     uint baudrate = spi_init(SD_SPI, m_sckfreq);
-    printf("SD SPI baudrate: %i\n", baudrate);
+    if (false) {
+      Trace::Log("SDCARD", "SD SPI baudrate: %i\n", baudrate);
```

Comment:
Better to do a Trace::Debug? otherwise lets remove the line

Reply:
Ooops I forgot to come back and make a new define in Cmake for this instead of just the `false`, I'll fix this.